algebras/testkit/src/main/scala/endpoints4s/algebra/client/ChunkedJsonEntitiesTestSuite.scala

```diff
@@ -68,17 +48,12 @@ trait ChunkedJsonEntitiesResponseTestSuite[T <: ChunkedJsonEntitiesTestApi]
     callStreamedEndpoint(streamedEndpointErrorTest, ())
       .map(_ shouldEqual expectedItems)
   }
-}
-
-trait ChunkedJsonEntitiesRequestTestSuite[T <: ChunkedJsonEntitiesTestApi]
-    extends ChunkedJsonEntitiesTestSuite[T] {
-  import streamingClient.{uploadEndpointTest}
 
-  "Encode chunks uploaded to a server" in {
+  "Succeed if server streams no chunks" in {
 
-    val expectedItems = List(Array[Byte](1, 2, 3), Array[Byte](4, 5, 6))
+    val expectedItems = List.empty
 
-    callStreamedEndpoint(uploadEndpointTest, expectedItems)
-      .map(_ => succeed)
+    callStreamedEndpoint(streamedEndpointEmptyTest, ())
+      .map(_ shouldEqual expectedItems)
   }
```

Comment:
There can never be enough tests for silly errors - noticed this one while playing around with Fetch `ChunkedEntities` 😄